### ISSUE TITLE
Illegal typescript generated when generating modules

### ISSUE THREAD

**Issue by lidermanrony** [bridgedotnet/Bridge#3181]
Typescript interfaces and vars aren't allowed to have dots '.' in them, however specifying the Module attribute generates such d.ts files - which renders them unusable.

### Steps To Reproduce

https://deck.net/b1857d505d7329994c1e3f11e6e7bb9e

```csharp
[Module(Name="someModule")]
public class Program
{
    public static void Main()
    {
        var msg = "Hello, World!";
        
        Console.WriteLine(msg);
    }
}
```

### Expected Result

```ts
/// <reference path="./bridge.d.ts" />

declare module someModule.Demo {
    export interface Program {
    }
    export interface ProgramFunc extends Function {
        prototype: Program;
        new (): Program;
        Main(): void;
    }
    var Program: ProgramFunc;
}

```

### Actual Result

```ts
/// <reference path="./bridge.d.ts" />

declare module Demo {
    export interface someModule.Program {
    }
    export interface someModule.ProgramFunc extends Function {
        prototype: someModule.Program;
        new (): someModule.Program;
        Main(): void;
    }
    var someModule.Program: someModule.ProgramFunc;
}
```

**Comment by geoffreymcgill** [comment]
Hi @lidermanrony, Thanks for submitting this issue. It's an interesting problem. We will investigate and fix.

**Comment by fabriciomurta** [comment]
The associated pull request has just been merged in. The fix will make it to Bridge 16.4.0.
